types/msg.go

```diff
@@ -6,10 +6,11 @@ import (
 
 //------- Results / Msgs -------------
 
-// HandleResult is the raw response from the handle call
+// HandleResult is the raw response from the handle call.
+// This is mirrors Rust's ContractResult<HandleResponse>.
 type HandleResult struct {
-	Ok  *HandleResponse `json:"Ok,omitempty"`
-	Err *StdError       `json:"Err,omitempty"`
+	Ok  *HandleResponse `json:"ok,omitempty"`
+	Err string          `json:"error,omitempty"`
```

Comment:
StdError is now removed, right?

Reply:
It is still in types, right? And actually used below with SystemError/QuerierResult.

Ugh, I don't have the current state of that refactor in my head. Anyway, if this is tested somehow (QuerierResult parsing by a contract), then I guess the code is good. But I am not sure if anything does test that until we hit wasmd

Reply:
Ah, thanks. In https://github.com/CosmWasm/go-cosmwasm/pull/139/commits/187f7d04d34ab44088400ab203e199f55bd91a69 I now removed StdError.